### ISSUE TITLE
... space filler makes it difficult to see -

### ISSUE THREAD

**Issue by foxsam21** [danielperna84/hass-configurator#70]
This is really a problem coming from the ace editor. The dots used to fill the spaces makes it difficult to see the all-important - in the yaml config files. The lines that start with a "  - " can easily be missed that they have or are missing it.
For now I just disabled "show invisibles".

**Comment by danielperna84** [comment]
That's not really an issue. As you point out, that's how the Ace Editor displays whitespace, which is useful to keep the syntax correct.
I guess there would be the possibility to add an option to manually set the color / transparency of whitespace. The default is:
```css
.ace-tm .ace_invisible {
	color: rgb(191, 191, 191);
}
```
If we change that to:
```css
.ace-tm .ace_invisible {
	color: rgba(191, 191, 191, 0.3);
}
```
the dots whould be less dominant. But that makes all other whitespace characters less visible as well. So doing this with a customizable option would be the way I would accept, leaving the default as it currently is.

**Comment by danielperna84** [comment]
I have pushed a commit to the dev-branch: [configurator.py](https://github.com/danielperna84/hass-configurator/blob/dev/configurator.py)
It's using the fixed 0.5 transparency. Is that enough for you to not miss "real" characters? If so, I'd go with that without adding support to set the transparency via setting.

**Comment by foxsam21** [comment]
The truth is I can't think of a better solution but the way it is now I feel like it is less than perfect. If someone can think of something better I think it will be an improvement. Especially for yaml files where a leading - can get swallowed up in those greyish dots that symbolise leading whitespace which is also necessary for those yaml files.

**Comment by danielperna84** [comment]
The thing is, that other popular text editors (I tested MS Word and Notepadd++) display these dots as well. And there the dots have been accepted for years. In Wort they are colored even more prominently. In Notepad++ they are faded out a little bit more. So besides making the markers less visible there is no other route that's acceptable, since when the big ones are doing it this way, most people are used to it. And in all the time my configurator is available this has been the first mention of this being a bad thing. So I guess it boils down to a matter of personal preference.
As said one post above, I pushed a new version to the dev branch. Replace your configurator.py with that one and tell me how that's for you. To me it's still visible enough to be useful, yet not as dominant anymore.
